Apps/WebClient/src/ClientApp/src/models/reportRequest.ts

```diff
@@ -0,0 +1,19 @@
+export default interface ReportRequest {
+    // Gets or sets the report data.
+    data: unknown;
```

Comment:
This looks like a nice place for inheritance since it could be a MedicationReportRequest that extends a ReportRequest